src/js/nav.js

```diff
@@ -0,0 +1,16 @@
+(function navegation() {
+  const btnOpen = document.querySelector('#open');
+  const body = document.querySelector('body');
+  const nav = document.querySelector('.wrapper-nav');
+  const btnClose = document.querySelector('#close');
+  function openNav() {
```

Comment:
todo esto se puede cubrir a puro CSS, y que desde javascript solo se revise por una clase. Por ejemplo, si al darle click al boton el elemento no tiene la clase is-open, entonces se le pone, y si ya la tiene se le quita, un toggle.

Ojo esto https://github.com/cetav-ddw/cegea/blob/develop/src/js/menu.js